operate/client/e2e-playwright/tests/processInstanceListeners.spec.ts

```diff
@@ -198,4 +201,43 @@ test.describe('Process Instance Listeners', () => {
     ).toBeVisible();
     await expect(page.getByText('Task listener', {exact: true})).toBeVisible();
   });
+
+  test('Listeners on process instance or participant (root flow node)', async ({
+    page,
+    processInstancePage,
+  }) => {
+    const processInstanceKey =
+      initialData.processWithListenerOnRootInstance.processInstanceKey;
+    await processInstancePage.navigateToProcessInstance({
+      id: processInstanceKey,
+    });
+
+    await processInstancePage.modifyInstanceButton.click();
+    await page.getByRole('button', {name: 'Continue'}).click();
+    await processInstancePage.diagram.clickFlowNode('Service Task B');
+    await page
+      .getByRole('button', {
+        name: 'Move selected instance in this flow node to another target',
+      })
+      .click();
+    await processInstancePage.diagram.clickEvent('End event');
+    await page.getByRole('button', {name: 'Apply Modifications'}).click();
+    await page.getByRole('button', {name: 'Apply', exact: true}).click();
```

Comment:
It works fine but is there a reason to modify the process here? If not, you might be able to get rid of it by removing the service task from the diagram (it's still a valid process which is immediately finished).

Reply:
Yes, we have an open [bug on the backend](https://github.com/camunda/camunda/issues/24744) side that causes a race condition between the creation of a process instance and a job for the listener, it's mapped out only for `start` type listeners at the moment, but to avoid any potential problems, I added a task so we have some "buffer" time and the instance doesn't immediately finish (which could also cause flakiness). This test will be simplified again as soon as said backend bug is fixed, eliminating the need for those modification steps.